docs/source/training_guide.rst

```diff
@@ -45,38 +51,38 @@ For the custom experiment you need a entrypoint function like this:
     from sklearn.model_selection import cross_val_score
     import os
     
-    # Função que fornece os passos para treinar o modelo com base em um conjunto de dados fornecido. 
-    # Seu nome (train_model) deve ser passado no campo 'training_reference'
-    # O seu parâmetro (base_path) deve conter o caminho de pastas para os arquivos que serão usados. 
-    # Um exemplo para o seu valor é: "/path/to/treino/customizado/experimento1"
+    # Function that provides the steps to train the model based on a given dataset.
+    # Its name (train_model) must be passed in the 'training_reference' field
+    # Its parameter (base_path) must contain the folder path for the files that will be used.
+    # An example for its value is: "/path/to/treino/customizado/experimento1"
     def train_model(base_path): 
 
-        ## Variáveis de ambiente carregadas de um arquivo fornecido pelo usuário no campo 'env'
+        # Environment variables loaded from a user-supplied file in the 'env' field
         # my_var = os.getenv('MY_VAR')
         # if my_var is None:
-        #    raise Exception("Could not find `env` variable")
+        #   raise Exception("Could not find `env` variable")
 
-        ## Variável de ambiente carregada do Neomaril com nome da base de dados (usado em alternativa a linha 61)
+        ## Environment variable loaded from Neomaril with database name (used alternatively to line 61)
         # df = pd.read_csv(base_path+'/'+os.getenv('inputFileName'))
-        df = pd.read_csv(base_path+"/dados.csv")    # Carrega a base de dados (dados.csv) que precisa ter o mesmo nome
-                                                    # arquivo enviado para o Neomaril no campo 'train_data'
+        df = pd.read_csv(base_path+"/dados.csv")    # Load the database (data.csv) which must have the same name
```

Comment:
Here should be "dados.csv" or you should change the parameter name that goes inside the `read_csv(...)`
Also... 
"# Load the database (data.csv) which must have the same filename
"# sent to Neomaril in the 'train_data' field